src/binder/fmt_impl.cpp

```diff
@@ -0,0 +1,6 @@
+#include "binder/expressions/bound_agg_call.h"
+#include "fmt/format.h"
+
+namespace bustub {
+auto BoundAggCall::ToString() const -> std::string { return fmt::format("{}({})", func_name_, fmt::join(args_, ", ")); }
```

Comment:
If this is the only one that you moved to .cpp, maybe we can move this back to .h?